poyoweb-server/src/auth/auth.ts

```diff
@@ -7,21 +8,24 @@ interface User {
   name: string;
 }
 
+const sleep = (ms: number) => new Promise<void>((resolve) => setTimeout(() => resolve(), ms));
+const errorSleep = sleep(2000);
```

Comment:
You should make the `errorSleep` inside the endpoint. Now it will only wait since 2000ms since booting the app. not from start of the http request